src/TodoApp.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import TodoList from './TodoList';
+import Footer from './Footer';
+
+class TodoApp extends React.Component {
+  state = {
+    todos: [],
+    inputValue: '',
+    allCompleted: false,
+    filterStatus: undefined,
+  }
+
+  addInputValue = (event) => {
+    this.setState({
+      inputValue: event.target.value,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    if (!this.state.inputValue) {
+      return;
+    }
+
+    this.setState((state) => {
+      const newTodo = {
+        title: this.state.inputValue,
+        id: +new Date(),
+        completed: false,
+      };
+
+      return {
+        todos: [...state.todos, newTodo],
+        inputValue: '',
+      };
+    });
+  }
```

Comment:
TodoApp не має обробляти івенти це не його завдання
```suggestion
  addTodo = (todo) => this.setState(prev => ({
    todos: [...prev.todos, todo],
    currentFilter: FILTER_TYPES.all,
  }))
```
Ось так вона має виглядати і нічого більше:)